using-timescaledb/update-db.md

```diff
@@ -31,10 +31,19 @@ ALTER EXTENSION timescaledb UPDATE;
 >:WARNING: When executing `ALTER EXTENSION`, you should connect using psql
 with the `-X` flag to prevent any .psqlrc commands from accidentally triggering
 the load of a previous DB version on session startup.
```

Comment:
this should probably read timescaledb version for more clarity

Reply:
Done